pkgs/development/python-modules/behave/default.nix

```diff
@@ -16,22 +16,21 @@ buildPythonApplication rec {
     sha256 = "1ssgixmqlg8sxsyalr83a1970njc2wg3zl8idsmxnsljwacv7qwv";
   };
 
-  checkInputs = [ mock pathpy pyhamcrest pytest pytest-html ];
+  checkInputs = [ mock pathpy pyhamcrest pytest pytest-html pytestCheckHook ];
   buildInputs = [ glibcLocales ];
   propagatedBuildInputs = [ colorama cucumber-tag-expressions parse parse-type six ];
 
   postPatch = ''
     patchShebangs bin
   '';
 
-  doCheck = true;
+  pytestFlagsArray = [ "tests/" ];
```

Comment:
Probably redundant b/c pytest would automatically find that folder, but not important.